src/sqlite-xamarin/sqlite-xamarin.targets

```diff
@@ -17,7 +17,7 @@
       <_ConfigureSqliteCommands
```

Comment:
Can we delete all of `sqlite-xamarin`? (doesn't have to be this PR)

This is for `Mono.Data.Sqlite.dll`? We don't even ship that in .NET 6+?

Reply:
Looks like we can do that, but indeed, definitely not in this PR. I'm trying to remove only the bare minimum here, that relates to libmonodroid.so